eve/main.yml

```diff
@@ -112,13 +124,87 @@ stages:
         worker: eve/workers/end2end
     steps:
     - Git: *git_pull
+    - SetProperties: *set_operator_properties
     - ShellCommand: *wait_docker_daemon
-    - ShellCommand:
+    - ShellCommand: *private_registry_login
+    - ShellCommand: &bootstrap_kind
         name: Bootstrap kind cluster
-        command: kind create cluster --config=kind-config.yaml
+        env:
+          <<: *kind-env
+        command: scripts/bootstrap-kind.sh ${KIND_NODE_IMAGE} ${VOLUME_ROOT}
         workdir: build/eve/workers/end2end
     - ShellCommand:
         name: Ensure cluster is up and running
         command: |
           kubectl get nodes
           kubectl get pods --all-namespaces
+    - ShellCommand:
+        name: Create namespace
+        command: kubectl create namespace ${STAGE} &> /dev/null
+    - ShellCommand: &create_image_pull_secret
+        name: Create pull image secret
+        command: scripts/create-pull-image-secret.sh | kubectl apply -f -
+        workdir: build/eve/workers/end2end
+        haltOnFailure: True
+    - ShellCommand:
+        name: Install operator
+        env:
+          <<: *kind-env
+          COMMIT_HASH: '%(prop:OPERATOR_IMAGE_TAG)s'
+        command: |
+          git clone ${OPERATOR_REPO} operator
+          cd operator
+          git checkout ${COMMIT_HASH}
+          docker pull ${OPERATOR_IMAGE}
+          kind load docker-image ${OPERATOR_IMAGE}
+          deploy/scripts/install-kind-deps.sh
+          deploy/scripts/install-zenko-operator.sh ${OPERATOR_IMAGE} "default"
+        workdir: build/eve/workers/end2end
+        haltOnFailure: true
+        flunkOnFailure: true
+    - ShellCommand:
+        name: Deploy Zenko Instance
+        env:
+          <<: *kind-env
+        command: scripts/deploy-zenko.sh "end2end" ${VOLUME_ROOT} "default"
+        workdir: build/eve/workers/end2end
+        haltOnFailure: true
+    - ShellCommand: &build-e2e-image
+        name: Build end2end test image
+        env: *e2e-env
+        command: |
+          docker build -t ${E2E_IMAGE_NAME}:${E2E_IMAGE_TAG} tests/zenko_tests
+          docker push ${E2E_IMAGE_NAME}:${E2E_IMAGE_TAG}
+          #kind load docker-image ${E2E_IMAGE_NAME}:${E2E_IMAGE_TAG}
+        workdir: build
+        haltOnFailure: true
```

Comment:
Can be done later IMO. But if you want to save some time here, this step can be translated into a stage, and the stage itself can be executed at the same time that  the documentation and iso are built. That way to pipeline looks like something like this:

```
---
   | --- build-doc
   | --- build-iso
   | --- build-e2e
---
|
---
    | --- end2end
---
```

It should make us save around 2 min of build time, not crazy but it's a simple win. Let's not do it now though, just an idea to keep in mind :)